public/index.html.erb

```diff
@@ -86,30 +86,33 @@
           <div id="register-bottom">
             <h4 class="bold-headline text-center" style="margin-top: 70px;">JOIN THE MOVEMENT</h4>
             <div class="teal-line center-block" style="margin-bottom: 30px;"></div>
-            <button class="slack-button center-block"><a class="link" href="https://slack.com/oauth/authorize?scope=bot&client_id=<%= ENV['SLACK_CLIENT_ID'] %>"><img src="img/slack_logo_button_@2x.png" style="width:40px; height:40px; margin-right:20px;"><span class="Add-MissingKids-Bot">Add MissingKids Bot to <span style="font-weight:600;">Slack</span></span></a></button>
+            <a class="link" href="https://slack.com/oauth/authorize?scope=bot&client_id=<%= ENV['SLACK_CLIENT_ID'] %>"><button class="slack-button center-block"><img src="img/slack_logo_button_@2x.png" style="width:40px; height:40px; margin-right:20px;"><span class="Add-MissingKids-Bot">Add MissingKids Bot to <span style="font-weight:600;">Slack</span></span></button></a>
           </div>
           <h4 class="bold-headline text-center" style="margin-top: 80px;">MISSINGKIDS BOT USES DATA FROM <a target='_blank' href='http://missingkids.org'>MISSINGKIDS.ORG</a></h4>
           <div class="teal-line center-block" style="margin-bottom: 30px;"></div>
         </div>
       </div>
       <div class="row">
-        <div class="col-lg-3 col-md-3">
+        <div class="col-lg-1 col-md-1">
         </div>
-        <div class="col-lg-6 col-md-6">
-          <p class="small-text text-center">According to the National Center for Missing and Exploited Children, roughly 800,000 children are reported missing each year in the United States -- that's roughly 2,000 per day.</p>
+        <div class="col-lg-10 col-md-10">
+          <p class="small-text text-center" style="font-size: 16px; font-weight: 700;">According to the National Center for Missing and Exploited Children, roughly 800,000 children are reported missing each year in the United States -- that's roughly 2,000 per day.</p>
         </div>
-        <div class="col-lg-3 col-md-3">
+        <div class="col-lg-1 col-md-1">
         </div>
       </div>
+	   <div class="row" style="margin-bottom: 10px;">
```

Comment:
If we're using CSS styles shouldn't be here. This is generally an issue elsewhere in the HTML too.